Podcast.xcodeproj/project.pbxproj

```diff
@@ -1097,7 +1097,7 @@
 			);
 			name = "[CP] Copy Pods Resources";
 			outputPaths = (
-				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}",
+				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/GoogleSignIn.bundle",
```

Comment:
is this from merge?

Reply:
I guess so, I didn't do anything

Reply:
Can I still merge the PR?

Reply:
Yea i don't think it will be an issue.. if it is we will revert